### ISSUE TITLE
🐞 Bug – App not properly generated in 0.19.x release if there is a syntax error in application code

### ISSUE THREAD

**Issue by projektove** [elm-land/elm-land#109]
## Problem

When having some syntax error in application code and I stop the elm-land server, then after I run a server again, an app is generated with Main.elm looking like this:

```
module Main exposing (main)

import Html exposing (Html)


main : Html msg
main =
    Html.text "Hello, Elm Land!"
```

and is not possible to fix the syntax error as there is no way how to see it.

## Your setup

Linux Debian 11
Node 18.16.1
Elm Land 0.19.1

**Comment by ryannhg** [comment]
Hi, @projektove – thanks for the bug report!

> and is not possible to fix the syntax error as there is no way how to see it.

Could you send me an example of the syntax error you are seeing? 

It would be helpful to see if the error is coming from a file in your `src/` folder, or from a file in the generated `.elm-land/src/` folder?

**Comment by projektove** [comment]
Maybe I'm not express the problem well. There are actually bugs in my application code, I know about them. The problem I have is, if I doesn't have more time to work this day and I stop the server in state where code doesn't compile, so when I start the server again next day, codegen create only that Main.elm from template, not real code which is needed for application to be functional and then is not possible to debug what is wrong in code.

I actually managed to fix all my bugs in code using `elm-lan build` command as you suggested, so now I'm not blocked. But IMO this behavior is wrong.

**Comment by ryannhg** [comment]
That makes sense, thank you for clarifying– I'll take a look at this issue and see if I can release a patch to address the problem!

The intended behavior is to make sure `elm-land server` is reporting errors clearly in the browser until all the problems are gone.

Thanks again for reporting this issue!

**Comment by sethlowie** [comment]
Integrating with elm-watch should resolve this, it generates default apps so you get something in the browser if it cannot generate the app do to compiler errors.

**Comment by ryan-haskell** [comment]
Good news– I just released [📦 Elm Land v0.20.0](https://github.com/elm-land/elm-land/releases/tag/v0.20.0) which resolves this issue (more details in the release linked above)

Thanks for improving Elm Land! 🌈
